### PR TITLE
Update KnotxServer default configuration to allow all *.html files be…

### LINKED ISSUE #269: Update KnotxServer default configuration to allow all *.html files
Current default configuration of `KnotxServer` supports only requests that request path match `/content/*` pattern.
Update default configuration, so that Knot.x with out-of-the-box configuration can support all html files (`*.html`).

### PR DESCRIPTION
**Pull request by Skejven**
Update KnotxServer default configuration to allow all (`*.`) GET requests.

Fixes #269 
---

I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
       "routing": {
         "GET": [
           {
-            "path": "/content/.*",
+            "path": ".*\\.html",
```
Comment (marcinczeczko) [inline knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json]: I think example app we can leave as it is now. Routes /content

Comment (Skejven) [inline knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json]: Ok, how about the tests configs?

Comment (marcinczeczko) [inline knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json]: I think. For test it doesn't matter

Comment (marcinczeczko) [inline knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json]: Up to you 😊

Comment (toniedzwiedz) [inline knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json]: The way I understand it, the previous config would accept query strings and the new one doesn't. Is my reasoning correct? If so, is this the desired behaviour?

Comment (Skejven) [inline knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json]: Good eye @toniedzwiedz , however in this case it works differently.
This path is used in [KnotxServerVertice](https://github.com/Cognifide/knotx/blob/16c6479cc8f47d83c1628dc0414df4c31460e676/knotx-server/src/main/java/io/knotx/server/KnotxServerVerticle.java#L58) to provide proper routing. When you check [Route.pathRegex](http://vertx.io/docs/apidocs/io/vertx/ext/web/Route.html#pathRegex-java.lang.String-) method documentation it says:
```
Set the path prefix as a regular expression. If set then this route will only match request URI paths, the beginning of which match the regex. Only a single path or path regex can be set for a route.
```

So the part important here is only a request path, not whole request. And this works fine with this prefix :)

Comment (tomaszmichalak) [inline knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json]: I am not sure that we should limit default routing to `.*\\.html`. You can have some other URL policy which breaks this rule. Someone can set rewrite rules like this for example: [http://knotx.io/hello-rest-service/](http://knotx.io/hello-rest-service/). In my opinion last routing entry should be `.*`.

File: knotx-server/src/main/resources/io.knotx.KnotxServer.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
       "routing": {
         "GET": [
           {
-            "path": "/content/.*",
+            "path": ".*\\.html",
```
Comment (tomaszmichalak) [inline knotx-server/src/main/resources/io.knotx.KnotxServer.json]: I am not sure that we should limit default routing to `.*\\.html`. You can have some other URL policy which breaks this rule. Someone can set rewrite rules like this for example: [http://knotx.io/hello-rest-service/](http://knotx.io/hello-rest-service/). In my opinion last routing entry should be `.*`.

Comment (toniedzwiedz) [inline knotx-server/src/main/resources/io.knotx.KnotxServer.json]: Ah, so this is where the default I was looking for comes from :)

**Comment by Skejven** [comment]
Updated default configuration, so that all GET requests are now processed.

**Pull request merged**